samples/client/petstore/python/tests/test_pet_api.py

```diff
@@ -222,7 +224,10 @@ def test_find_pets_by_status(self):
 
     def test_find_pets_by_tags(self):
         self.pet_api.add_pet(self.pet)
-
+        with open("file_out.txt", "w") as fout:
```

Comment:
@sebastien-rosset why are we writing to disk here?
It looks like this may be for debugging on your machine only. If it is, can you delete it?

Reply:
yes this was for debugging, I am removing.